becquerel/core/peakfinder.py

```diff
@@ -293,7 +293,7 @@ def add_peak(self, xpeak):
 
     def plot(self, facecolor="red", linecolor="red", alpha=0.5, peaks=True):
         """Plot the peak signal-to-noise ratios calculated using the kernel."""
-        bin_edges = self.spectrum.bin_edges_raw
+        self.spectrum.bin_edges_raw
```

Comment:
This again looks like an autoformatter change. I'm not sure I like it... I would prefer a notification that a variable is unused instead.

Reply:
I am not following why you want a notification that a variable is unused. An unused var creates an extra reference in memory which is not used below so it doesn't make sense to track it. This is a flake8 error (pep something or another) and not removed by black

Reply:
Right, we should just delete the line entirely instead.

Reply:
Good point. I had originally read it as executing a function. Since this is a property which just returns a hidden attribute I've remove it: 20c093d501e0d6cb86bd41c0be6a3243be2bf2b9